lib/reporters/junit.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Author: Josh Hagins
```

Comment:
I don't have a Jenkins install handy, but I have a screenshot from Bamboo on a test project that we have here. 

In this case, it picks up test results from the reporter output. If there are no validation errors there is no reporter file so no test case results. But that's okay as the task has succeeded anyway.

I thought about extending the reporter to output something even if validation passes, but it's a bit redundant for this use case and that wouldn't be consistent with the behaviour of the other grunt-html reporters anyway. 

![screen shot 2016-06-16 at 2 54 41 pm](https://cloud.githubusercontent.com/assets/585574/16106121/fc90474a-33d4-11e6-9d13-e46844fbe6b1.png)

![screen shot 2016-06-16 at 2 48 45 pm](https://cloud.githubusercontent.com/assets/585574/16105972/675232fc-33d3-11e6-99cd-913efc6319b6.png)